godot_project/editor/controls/dockers/workspace_docker/simulations_docker/simulations_docker_controls/relax_tools_panel.tscn

```diff
@@ -61,13 +61,6 @@ layout_mode = 2
 button_pressed = true
 text = "Treat incomplete valences as if they have been filled by hydrogens"
 
-[node name="LabelSelectOnlyNotice" type="Label" parent="."]
-unique_name_in_owner = true
-layout_mode = 2
-text = "ℹ When relaxing selected molecules, all atoms in the molecule should be selected"
-horizontal_alignment = 1
-autowrap_mode = 2
-
```

Comment:
Maybe also rename "Only selected molecules" by "Selection only", ( the checkbox inside `PanelContainer/VBoxContainer` )

Reply:
Good thinking